js/__fixtures__/did-key-ed25519.json

```diff
@@ -0,0 +1,612 @@
+[
+  {
+    "seed": "4f66b355aa7b0980ff901f2295b9c562ac3061be4df86703eb28c612faae6578",
+    "keypair": {
+      "application/did+json": {
+        "id": "#z6Mkk7yqnGF3YwTrLpqrW6PGsKci7dNqh1CjnvMbzrMerSeL",
+        "type": "JsonWebKey2020",
+        "controller": "did:key:z6Mkk7yqnGF3YwTrLpqrW6PGsKci7dNqh1CjnvMbzrMerSeL",
+        "publicKeyJwk": {
+          "crv": "Ed25519",
+          "x": "VDXDwuGKVq91zxU6q7__jLDUq8_C5cuxECgd-1feFTE",
+          "kty": "OKP"
+        },
+        "privateKeyJwk": {
```

Comment:
Apart of me thinks it's probably a bad idea to include private key info in a security-vocab definition because people will likely copy the style and think it's acceptable. Off the top of my head I don't know how else to deterministically handle tests without mocking private material (seeds/private keys). I'll have to get back to this with a suggestion on another way to go with this.

Reply:
https://tools.ietf.org/html/rfc7517#appendix-A.2

I think its a mistake NOT to include private keys, when trying to explain how cryptography works.

Reply:
I removed the did key test vectors from the project, but I refuse to remove private keys from tests, if they are needed to write the test :)